README.md

```diff
@@ -9,5 +9,30 @@ organisation of neuroimaging data.
 In this repository, we develop the
 [BIDS specification](https://bids-specification.readthedocs.io/en/latest/).
 
-If you want to become a part of the BIDS community and contribute to the specification, please see our
-[contributing guidelines](./CONTRIBUTING.md).
+# Contributing to BIDS
```

Comment:
I feel like this section is mixing a README and a CONTRIBUTING guide. [Here's an example](https://github.com/me-ica/tedana#getting-involved) of how I'd think about this section, with specific suggestions below :point_down: 

```suggestion
# Contributing to BIDS 

BIDS is a community-driven standard, and it depends on contributions from its users.
We :yellow_heart: new contributors, and we appreciate all contributions to the project!
To see a current list of our contributors, please see our [Contributors appendix](https://github.com/bids-standard/bids-specification/blob/master/src/99-appendices/01-contributors.md). 

When you're ready to get started, check out [our contributing guidelines](https://github.com/bids-standard/bids-specification/blob/master/CONTRIBUTING.md).

Want to learn more about working with BIDS? Have a question, comment, or suggestion?
Open or comment on one of our [NeuroStars issues](https://neurostars.org/tags/bids)!

We ask that all contributors to BIDS across all project-related spaces (including but not limited to:
GitHub, the [Google group](https://groups.google.com/forum/#!forum/bids-discussion), and newsletter emails),
adhere to our [code of conduct](https://github.com/bids-standard/bids-specification/blob/master/CODE_OF_CONDUCT.md).
```

Reply:
Line 14 of @emdupre proposal - (users -> members)

```suggestion
BIDS is a community-driven standard, and it depends on contributions from its members.
```